doc/json-schema/os-list-schema.json

```diff
@@ -298,6 +300,18 @@
                                     "systemd"
                                 ]
                             },
+                            "devices": {
+                                "$id": "#/properties/os_list/items/anyOf/0/properties/compat_with",
+                                "type": "array",
+                                "title": "The compat_with schema",
+                                "description": "Provides a JSON-format list of strings representing Raspberry Pi devices that are supported with this image",
+                                "default": "",
+                                "examples": [
+                                    "[\"1a\", \"1b\"]",
```

Comment:
One of the examples here is `"1a"`, but on line 198 you've used `"pi1a"` :facepalm: 
Probably makes sense to have a list somewhere of all the possible values that can be used here, for the benefit of OS-maintainers who'll be adding this to their `os.json` files?

Reply:
>Probably makes sense to have a list somewhere of all the possible values that can be used here, for the benefit of OS-maintainers who'll be adding this to their os.json files?

In the original design the list can simply be seen in the top os.json file:

```
    "imager": {
        "latest_version": "1.7.4",
        "url": ["https://www.raspberrypi.org/software/"](https://www.raspberrypi.org/software/),
        "devices":
        [
            {"name": "Raspberry Pi Zero", "tags": ["pi0"]},
            {"name": "Raspberry Pi Zero 2", "tags": ["pi02", "64b"]},
            {"name": "Raspberry Pi 1", "tags": ["pi1"]},
            {"name": "Raspberry Pi 2", "tags": ["pi2"]},
            {"name": "Raspberry Pi 3/3+", "tags": ["pi3", "64b"]},
            {"name": "Raspberry Pi 4", "tags": ["pi4", "64b"], "default": true}
        ]
    },
    "os_list": [

        {
            "name": "Raspberry Pi OS (64-bit)",
            "description": "A port of Debian Bullseye with the Raspberry Pi Desktop (Compatible with Raspberry Pi 3/4/400)",
            "icon": ["https://downloads.raspberrypi.org/raspios_arm64/Raspberry_Pi_OS_(64-bit).png"](https://downloads.raspberrypi.org/raspios_arm64/Raspberry_Pi_OS_(64-bit).png),
            "url": ["https://downloads.raspberrypi.org/raspios_arm64/images/raspios_arm64-2023-05-03/2023-05-03-raspios-bullseye-arm64.img.xz"](https://downloads.raspberrypi.org/raspios_arm64/images/raspios_arm64-2023-05-03/2023-05-03-raspios-bullseye-arm64.img.xz),
            "extract_size": 4412407808,
            "extract_sha256": "962780be6bb41522532f26449f67524dc61038673833c079808da9ca2ad9a4f0",
            "image_download_size": 858074108,
            "release_date": "2023-05-03",
            "init_format": "systemd",
            "devices": ["64b"]
        },
        {
            "name": "Raspberry Pi OS (32-bit)",
            "description": "A port of Debian Bullseye with the Raspberry Pi Desktop (Recommended)",
            "icon": ["https://downloads.raspberrypi.org/raspios_armhf/Raspberry_Pi_OS_(32-bit).png"](https://downloads.raspberrypi.org/raspios_armhf/Raspberry_Pi_OS_(32-bit).png),
            "url": ["https://downloads.raspberrypi.org/raspios_armhf/images/raspios_armhf-2023-05-03/2023-05-03-raspios-bullseye-armhf.img.xz"](https://downloads.raspberrypi.org/raspios_armhf/images/raspios_armhf-2023-05-03/2023-05-03-raspios-bullseye-armhf.img.xz),
            "extract_size": 4143972352,
            "extract_sha256": "b68288da5133e8acf87a656f8a03526ef5cb8b7d7521833ef9bfe5df832e7346",
            "image_download_size": 914230732,
            "release_date": "2023-05-03",
            "init_format": "systemd",
            "devices": ["pi0","pi1","pi2"]
        },
```

Not sure that's still the plan, as I don't see it mentioned in schema.

Reply:
Ahhh, I didn't think to look there.